front-end/src/app/layout/committee-banner/committee-banner.component.ts

```diff
@@ -46,28 +48,26 @@ export class CommitteeBannerComponent implements OnInit, OnDestroy {
     Z: 'national party non-federal account',
   };
 
-  public active_committees = ['M', 'Q', 'W', 'D'];
+  public activeCommittees = ['M', 'Q', 'W', 'D'];
```

Comment:
Maybe since you're already renaming it, we could call it `activeStatuses` since it's a list of status codes rather than committees

Reply:
This variable name was updated to activeStatusCodes